README.md

```diff
@@ -1,3 +1,94 @@
 # Language Features a la carte
 
 A tool checking that some specific language features are not used
+
+## How to use the tool
+Defining which constructs are allowed to be used in a program can be done in two ways:
+* Blacklist-based checking: specify the constructs that <b>should not</b> be used
+* Whitelist-based checking: specify the constructs that <b>are allowed</b> to be used
+
+### Blacklisting language features
+To check a program using a blacklist-based approach, instantiate a `BlacklistChecker` with
+`BlacklistRule`s that match the constructs that should not be allowed.
+`BlacklistRule`s can be either predefined ones, available in `BlacklistRules` (e.g.
+`BlacklistRules.NoVar`) or custom ones.
+
+E.g. to check that a program does not use `while` or `var`:
+```Scala
+import syntactic_checker.blacklist.{ BlacklistChecker, BlacklistRules }
+import scala.meta.dialects.Sbt1
```

Comment:
Not very important, but does that work with the Scala 3 dialect instead of sbt? Sbt is not a proper Scala dialect.

Reply:
It indeed works in Scala 3. Changed in e94188bfd846e08dfee286cf990e1a69dae1e92f